routes/celebrities.routes.js

```diff
@@ -0,0 +1,44 @@
+/* const CelebrityModel = require("../models/Celebrity.model"); */
+
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
+
+const Celebrity = require('../models/Celebrity.model');
+
+// all your routes here
+
+module.exports = router;
```

Comment:
This works but as as good practice we always put our module.exports at the end of the file